esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2088,6 +2088,8 @@ bool CFileSprayEx::onSprayFixed(IEspContext &context, IEspSprayFixed &req, IEspS
         if (!req.getExpireDays_isNull())
             options->setExpireDays(req.getExpireDays());
 
+        options->setNoCommon(req.getNoCommon());
```

Comment:
This line shouldn't be needed.
There is already code to set it on line 2080.

Reply:
Removed